documentation/src/main/asciidoc/topics/proc_adding_embeddedcm_bean.adoc

```diff
@@ -1,7 +1,6 @@
 [id='adding-embeddedcm-bean_{context}']
 = Adding the EmbeddedCacheManager Bean
-The {brandname} starter operates in Remote Client/Server mode with `infinispan-spring-boot-starter-remote` on the classpath by default.
-To use embedded caches, you need to add the `EmbeddedCacheManager` bean.
+Configure your application to use embedded caches.
```

Comment:
@danberindei I've tried to simplify this by dropping the details about operating in Remote Client/Server mode, which is terminology we should try to get away from anyway.

Reply:
:+1: to get rid of as many "modes" as possible (degraded mode, especially when spelled DEGRADED_MODE, is an old pet peeve of mine).